model/riscv_sys_regs.sail

```diff
@@ -153,6 +153,15 @@ val sys_enable_next = {c: "sys_enable_next", ocaml: "Platform.enable_next", _: "
 /* Whether FIOM bit of menvcfg/senvcfg is enabled. It must be enabled if
    supervisor mode is implemented and non-bare addressing modes are supported. */
 val sys_enable_writable_fiom = {c: "sys_enable_writable_fiom", ocaml: "Platform.enable_writable_fiom", _: "sys_enable_writable_fiom"} : unit -> bool
+
+/* How many PMP entries are implemented. This must be 0, 16 or 64 (this is checked at runtime). */
+val sys_pmp_count = {c: "sys_pmp_count", ocaml: "Platform.pmp_count", _: "sys_pmp_count"} : unit -> range(0, 64)
+/* Which PMP entries are writable. This is a bitmask. 1 means writable, 0 means read-only zero. */
```

Comment:
As above: reset value is not required to be zero, regardless of whether the PMP register is readonly or not. Also, it is possible and legal for some bits within a PMP_addr to be writable (and even have specific legal values)

Reply:
Will Sail complain if no reset value is defined and someone tries to read it?

Reply:
For a register that has no initial value, it would be treated as an unknown value, which could be any possible value of the register type. That isn't really observable unless you use symbolic execution though.

Here I think these values are coming from external `sys_` functions which means the configuration/emulator/environment must provide a concrete value to the model.